createNginx.sh

```diff
@@ -14,6 +13,7 @@ NGINX_MAX_UPLOAD_SIZE=${NGINX_MAX_UPLOAD_SIZE:-200m}
 LDAP_NAME=${8:-openldap}
 LDAP_DOMAIN=${9:-demo.com}
 LDAP_PASSWD=${10:-secret}
+PHPLDAPADMIN_NAME=${11:-phpldapadmin}
```

Comment:
Try to keep the arguments in a sane order. So maybe throw PHPLDAPADMIN_NAME after NEXUS_NAME to keep all of those consolidated near each other instead of at the bottom? That includes the command line argument ordering.